i18n/pl.json

```diff
@@ -144,7 +143,7 @@
     "PROFILE_INVALID_WEBSITE": "Podałeś nieprawidłowy link! Przykłady prawidłowych linków: https://example.com, http://example.com, example.com\nMaksymalna długość linku: 25 znaków!",
     "PROFILE_LOCATION": "Lokalizacja",
     "PROFILE_NOT_SET": "Nie ustawiono!",
-    "PROFILE_NOT_SET_DESCRIPTION": "Nie ustawiono! Ustaw swój profil za pomocą /profile set",
+    "PROFILE_NOT_SET_DESCRIPTION": "Nie ustawiono! Ustaw swój profil za pomocą /profil ustaw",
```

Comment:
this should use slash command mentions

Reply:
that string isnt rendered inside discord, its the one that shows up in the /profile view image